### PR TITLE
Zmenu updates

### PR DESCRIPTION
**Pull request by azangru**
## Description

- Include gene and transcript location in zmenu payload
- Add "Mane Plus Clinial"
- Do not prettify biotypes by passing them through the small values list; show them as they are
- Show "Mane Select" consistently at different zoom levels
- Change `canonical` to `Ensembl canonical`

## Jira tickets
- https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1974
- https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2158
- https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2139
- https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2281

## Review deployment url
http://gb-zmenu-updates.review.ensembl.org

### CODE REVIEW

File: backend-server/config/small-values.toml [hunk 1]
```diff
@@ -116,6 +116,8 @@ unprocessed_pseudogene="unprocessed pseudogene"
 vault_rna="vault RNA"
 
 [designation.label]
```
Comment (veidenberg) [inline backend-server/config/small-values.toml:118]: Can we just keep `[designation.label]` section and clean the rest of the file (as it's the only field used)?

File: backend-server/egs-data/egs/v16/gene/gene-common-zmenu.eard [hunk 2]
```diff
@@ -132,14 +160,18 @@ struct!(gene_tr_zmenu,"""
       gene.desigtr_unversioned,
       gene.id_versioned,
       index(["gene-other-rev","gene-pc-rev","gene-other-fwd","gene-pc-fwd","focus"],gene.leaf_index),
-      small_value("gene-biotype","label",gene.biotype)
+      gene.biotype,
+      gene_location,
+      small_value("designation", "label", gene.designation)
```
Comment (veidenberg) [inline backend-server/egs-data/egs/v16/gene/gene-common-zmenu.eard:165]: I guess we cannot use raw values here?

File: backend-server/config/small-values.toml (hunk 1 above)
Comment (azangru) [reply backend-server/config/small-values.toml:118]: Yes, good point. I wanted to ask on Friday whether we should; and then forgot.

File: backend-server/egs-data/egs/v16/gene/gene-common-zmenu.eard (hunk 2 above)
Comment (azangru) [reply backend-server/egs-data/egs/v16/gene/gene-common-zmenu.eard:165]: That's saved as mane_select, mane_plus_clinical, canonical, or "normal" (which is then ignored). In the right-hand sidebar and in entity viewer we write "MANE Select" and "MANE Plus Clinical". Steve and Andrea want consistency.

**Review by veidenberg — APPROVED** [review]

**Pull request merged**